src/routes/dictionaries/+page.svelte

```diff
@@ -35,7 +35,11 @@
   let isSequenceAdaptation: boolean = false;
   let sequenceAdaptationName: string;
 
-  $: hasCreatePermission = featurePermissions.commandDictionary.canCreate(data.user);
+  $: hasCreatePermission =
+    featurePermissions.channelDictionary.canCreate(data.user) &&
+    featurePermissions.commandDictionary.canCreate(data.user) &&
+    featurePermissions.parameterDictionary.canCreate(data.user) &&
+    featurePermissions.sequenceAdaptation.canCreate(data.user);
   $: createButtonDisabled = !files || files?.length === 0 || (isSequenceAdaptation && sequenceAdaptationName === '');
```

Comment:
it seems like if you create a dictionary, the button still doesn't get disabled despite the input being reset.